runner/runner.go

```diff
@@ -0,0 +1,168 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"sync/atomic"
+	"time"
+
+	"github.com/cybozu-go/github-actions-controller/agent"
+	"github.com/cybozu-go/github-actions-controller/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Runner interface {
+	Run(ctx context.Context) error
+	DeletionTimeHandler(w http.ResponseWriter, _ *http.Request)
+	MetricsInit()
+}
```

Comment:
Why do you use `interface`?

Reply:
There was no particular reason for this, so I changed it to Runner struct.